mkdocs/docs/getting-started/your-first-experiment.md

```diff
@@ -48,54 +48,15 @@ cd load-test-http
 ```
 
 ## 3. Run experiment
-We will load test and validate the HTTP service whose URL is https://example.com. We will specify that the error rate must be 0, the mean latency must be under 50 msec, the 90th percentile latency must be under 100 msec, and the 97.5th percentile latency must be under 200 msec. 
+We will load test the HTTP service whose URL (`url`) is https://httpbin.org/get. 
 
-Run the experiment as follows.
+The `iter8 run` command combines an experiment chart with values, generates the `experiment.yaml` file, runs the experiment, and writes results into the `result.yaml` file. Run the experiment as follows.
 
 ```shell
-iter8 run --set url=https://example.com \
-          --set SLOs.error-rate=0 \
-          --set SLOs.latency-mean=50 \
-          --set SLOs.latency-p90=100 \
-          --set SLOs.latency-p'97\.5'=200
+iter8 run --set url=https://httpbin.org/get
```

Comment:
I wonder if we've made this too simple by removing the SLOs. I believe SLOs are a core feature (see "Features at a glance"). I believe we should have at least one SLO in this example. It does not make it conceptually more difficult, IMO.

Reply:
I agree SLOs are a core feature.

However, the first experiment does provide value without SLOs already. In particular, it is benchmarking an application and reporting metrics information.

***

With v0.9, let us explore your second experiment along the following lines... 

```yaml
- your second experiment
  - with SLOs # with some basic description of what SLOs are
  - inside GitHub Actions
  - inside Tekton
  - with notifications
  - with rollouts
  - inside Kubernetes
  - with AutoX
```

All of the above can be part of the getting started page as well ... idea is they are mostly agnostic to the experiment type (load-test-http, load-test-grpc, mirroring, etc.)

Reply:
this is great. i don't understand why we are cutting everything out in advance of that. between now and then users have to piece it together on their own. i see greater value in the simple example that includes slos

Reply:
Your first experiment is not meant to teach everything about Iter8 in one go. It is meant to get the user to run the simplest possible experiment to get started with Iter8. The section is titled "getting started", and this is the simplest possible start.
***
The idea is to keep your first experiment, as simple as possible ...

Reply:
ok. i see a couple of next steps that attracted my attention. i think this is good